src/transform/inverse.rs

```diff
@@ -1492,7 +1492,9 @@ fn av1_idct64(input: &[i32], output: &mut [i32], range: usize) {
   output[63] = clamp_value(stg10[0] - stg10[63], range);
 }
 
-static INV_TXFM_FNS: [[fn(&[i32], &mut [i32], usize); 5]; 4] = [
+type InvTxfmFn = fn(&[i32], &mut [i32], usize);
```

Comment:
```suggestion
type InvTxfmFn = fn(input: &[i32], output: &mut [i32], bits_size: usize);
```

While at it.

Reply:
Done, but instead of `bits_size`, I used `range` to match signatures of inverse transform functions. I also think that it's a better name, because it's more clear that it's a clamping range.